### ISSUE TITLE
Thread 1: signal SIGABRT

### ISSUE THREAD

**Issue by vvscode** [transistorsoft/react-native-background-geolocation#983]
## Your Environment
* Plugin version:
```
// package.json
"react-native-background-geolocation": "^3.4.2",
// Podfile.lock
RNBackgroundGeolocation: 8f0a20788100fbf08987d80f2053ae6eba79c566
```
* Platform: iOS
* OS version: iOS 13.2
* Device manufacturer / model: iPhone 8
* React Native version (`react-native -v`):
* Plugin config
```javascript <!-- Syntax-highlighting:  paste your code below -->
  return BackgroundGeolocation.ready({
    // Don't run the background-service when app is closed.
    stopOnTerminate: true,
    startOnBoot: false,
    // Android minimum interval is 60 seconds.
    // It is impossible to have a heartbeatInterval faster than this on Android.
    heartbeatInterval: 60,
    // [iOS only] Prevent iOS from suspending your application in the background.
    // Required to make heartbeatInterval work in iOS.
    preventSuspend: true,
    // The minimum distance (measured in meters) a device must move horizontally
    // before an update event is generated.
    distanceFilter: 0,
    // Use of foreground service should ensure the system doesn't kill it when low on memory.
    // Defaults to true on Android >= 8.0 (but we have users on older devices as well).
    foregroundService: true,
  });
```

## Expected Behavior
It should start listening to the location updates

## Actual Behavior
It gives me `Thread 1: signal SIGABRT` on starting 
![image](https://user-images.githubusercontent.com/6904368/74716376-16f42780-5237-11ea-8cc7-07c175995810.png)
![image](https://user-images.githubusercontent.com/6904368/74716401-23788000-5237-11ea-9a7d-834f773c8b91.png)

## Steps to Reproduce

1.
2.
3.
4.

## Debug logs

<details><summary>Logs</summary>

``` <!-- Syntax highlighting:  DO NOT REMOVE -->
PASTE_YOUR_LOGS_HERE
```

</details>

**Comment by vvscode** [comment]
Discovered that it's caused by updating `react-native-background-fetch` to `2.8.0` - so this is mostly a warning to those, who did the same

**Comment by christocracy** [comment]
I updated the [Setup Docs](https://github.com/transistorsoft/react-native-background-geolocation/blob/master/help/INSTALL-IOS-AUTO.md#ios-auto-linking-installation) to pin `2.7.1`.

In a perfect world, one wouldn't need to do this manual `yarn add react-native-background-fetch`, since its defined as a [`dependency` in `react-native-background-geolocation`](https://github.com/transistorsoft/react-native-background-geolocation/blob/master/package.json#L47), but RN does not auto-link child dependencies.

`react-native-background-fetch@2.8.0` (now released as `3.0.0`) is a major APi refactor and existing version of `background-geolocation` is only compatible with the version defined in its `package.json` (`~2.7.1`).

Thus, one needs to pin to `2.7.1`
```
yarn add react-native-background-fetch@2.7.1
```

**Comment by christocracy** [comment]
`background-geolocation` support for `react-native-background-fetch@3.0.0` is due to be released soon.
